test/unit/l10n_test.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:math';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:messenger/l10n/l10n.dart';
+
+void main() {
+  test('formattedFileSize()', () async {
+    WidgetsFlutterBinding.ensureInitialized(); // needed to load translations
+    await L10n.set(Language.fromTag('en-US'), refresh: false);
```

Comment:
Oh, you may use there:

```suggestion
    await L10n.init(L10n.languages.first);
```

This `init()` method will also initialize the date formatting, just in case.

Reply:
I've applied your suggestion